src/deploy/deploy.py

```diff
@@ -273,44 +273,32 @@ def _confirm_deploy(project, release, environment_id, wait_for_seconds, interval
     start_timer = time.perf_counter()
     release_id = release["release_id"]
 
-    def _is_release_deployed():
-        is_release_deployed = project.is_release_deployed(release, environment_id)
-        waited_for_seconds = int(time.perf_counter() - start_timer)
-
-        if not is_release_deployed and waited_for_seconds < wait_for_seconds:
-            click.echo(
-                click.style(
-                    f"Trying again in {interval}s, (waited {wait_for_seconds}s).",
-                    fg="yellow"))
-            time.sleep(interval)
-            _is_release_deployed()
-
-        if not is_release_deployed and waited_for_seconds >= wait_for_seconds:
-            return False
-
-        if is_release_deployed:
-            return {
-                'waited_for_seconds': waited_for_seconds
-            }
-
     click.echo("")
     click.echo(click.style(f"Checking deployment of {release_id} to {environment_id}", fg="yellow"))
     click.echo(click.style(f"Allowing {wait_for_seconds}s for deployment.", fg="yellow"))
 
-    confirmation = _is_release_deployed()
+    while not project.is_release_deployed(release, environment_id):
+        waited_seconds = int(time.perf_counter() - start_timer)
```

Comment:
I changed the name of this variable in my other PR to `total_time_waited` – so the variables are more different, and less likely to be confused.